source/CardGames.py

```diff
@@ -235,23 +260,169 @@ def printHands(self, players: dict):
+    self.info_player_asked = self.playerinfo[self.index(player_asked, self.playerinfo)][1]
+    self.player_turn = self.playerinfo[self.index(player_turn, self.playerinfo)][0]
+    self.player_asked = self.playerinfo[self.index(player_asked, self.playerinfo)][0]
+    turn_value = self.sayGoFish()
+      
+    self.surrender_card()
+
+    return turn_value
+    
+  def confirm_value(self):
+    confirm_value = True
+    while confirm_value:
+      confirm = input("is this correct? Y/N: ")
+      if confirm == "n":
+        num_players = int(input("How many players? "))
+      elif confirm == "y":
+        confirm_value = False
+      else:
+        print("\n[Please input Y/N]\n")
+          
+  def multiplayer_info(self):
+    width = os.get_terminal_size().columns
+    print()
+    print("*-"*(width//2))
+  
+    num_players = int(input("How many players? "))
+
+    print()
+
+    print("%d players selected" %num_players, "\n")
+
+    self.confirm_value()
+        
+    for i in range(1, num_players+1):
+      print()
+      player = input("Player %d What is your name? " %i)
+      self.players.append(Player(player))
+
+    print()
+    print("*-"*(width//2))
+    print("[ TOTAL NUMBER OF PLAYERS ]".center(width))
+    print(str(num_players).center(width))
+
+    return self.assign_player_numbers()
+
+  def assign_player_numbers(self):
+    player_num = {}
+
+    youngest_player = self.playerages()
+    self.players.insert(0, self.players.pop(self.players.index(youngest_player)))
+
+    for counter, player in enumerate((self.players), start =1):
+      print("PLAYER %d: " % counter, player)
+      player_num[counter] = player
+    print()
+    self.player_dict = player_num
+    return player_num
+      
+  def point_tracker_interface(self, info):
+    width = os.get_terminal_size().columns
+    score = 0
+
+    print("*-"*(width//2))
+    
+    print("[ PLAYER SCORES ]".center(width))
+    for num, player in info.items():
+      print(("{0}: {1}".center(width)).format(player, score))
+
+    print()
+    print("*-"*(width//2))
+
+
+  def count_cards(self):  #How many cards are in the deck assigned to "num_cards"  
+      num_cards = self.deck.size
+      print()
+      print(" "*25, "*"*5)
+      print("cards remaining in deck: ", num_cards)
+      print(" "*25, "*"*5)
+      print()
+      #For each iteration the number of cars in the deck decrease by 1 card  
+      if num_cards == 0:
+        print("The deck is now empty. Game over, Thanks for playing!")
+        self.game_over()
+        return True
+
+
+
+  def game_over(self):
+    width = os.get_terminal_size().columns
+    deco = "*-"*(width//2)
+    j = 0
+    gm = ["G", "A", "M", "E", "O", "V", "E","R"]
+    print(deco)
+    for i in range(width//8):
+      print("  "*j, ". *")
+      j += 3
+      if i == 10:
+        for i in range (len(gm)):
+          print("  "*j, gm[i])
+          j += 3
+    print(deco)
+    sys.exit()
+
```

Comment:
I love the way point_tracker works and the count_cards function is well-made. I like the end condition setup!

Reply:
Thank you, it can definitely be improved, but I appreciate your feedback!